### ISSUE TITLE
[Bug Report][3.3.10] VListItem missing anchor title

### ISSUE THREAD

**Issue by bogdanciuca** [vuetifyjs/vuetify#17908]
### Problem to solve
Similar to `VBtn` and `VTab`, `VListItem` can be used as an anchor. Due to the fact that `title` is already used as a prop for the component, it can't be used as an anchor title, thus limiting accessibility and SEO.

### Proposed solution
Define another prop for `VListItem`, e.g. `linkTitle` or `anchorTitle` and assign its value to the tag's `title` property (when the component is rendered as a link).

### Accepted solution
Use item's `title` prop instead, see [discussion](https://github.com/vuetifyjs/vuetify/pull/17909) for further details.

**Comment by KaelWD** [comment]
I brought this up when we added the title prop, if you want title as an attribute you have to use `:title.attr="'Title'"` instead.

**Comment by johnleider** [comment]
> I brought this up when we added the title prop, if you want title as an attribute you have to use `:title.attr="'Title'"` instead.

Add it to the docs

**Comment by bogdanciuca** [comment]
@KaelWD thanks for the suggestion. However, it seems that `title.attr` doesn't work with SSG / SSR, only with CSR. I've opened a [follow up issue](https://github.com/vuetifyjs/vuetify/issues/18457).
